cmake/klee_add_test_executable.cmake

```diff
@@ -1,27 +1,44 @@
-#===------------------------------------------------------------------------===#
+#===-- klee_add_test_executable.cmake --------------------------*- CMake -*-===#
 #
 #                     The KLEE Symbolic Virtual Machine
 #
 # This file is distributed under the University of Illinois Open Source
 # License. See LICENSE.TXT for details.
 #
+# Author: Ian Glen Neal
+#
+# Purpose: Add a new target that is compiled into a native executable and 
+#          has it's bitcode automatically extracted and copied into the 
+#          bin/ folder. This is useful for compiling full benchmarks which
+#          can be immediately run in KLEE without having to manually invoke
+#          'extract-bc' after recompilation. A convenience facility.
+#          NOTE: These executables cannot contain calls to klee_* functions,
```

Comment:
Really? Not even if that application `#include "klee.h"`?

Reply:
The `klee_*` family of functions aren't real functions. They're just intercepted by the SpecialFunctionHandler during symbolic execution. You could include "klee.h", but if you actually tried to invoke `klee_make_symbolic` and compile into a native executable, there would be nothing to link with. That's why KLEE examples always have you compile into bitcode and not full executables.